python/dgl/graphbolt/item_sampler.py

```diff
@@ -184,83 +184,18 @@ def __iter__(self):
-                not self._drop_last and last_batch_count == self._num_replicas
-            ):
-                # No need to drop uneven batches.
-                num_evened_items = num_items
-                if num_workers > 1:
-                    total_batch_count = (
-                        num_items + self._batch_size - 1
-                    ) // self._batch_size
-                    split_batch_count = total_batch_count // num_workers + (
-                        worker_id < total_batch_count % num_workers
-                    )
-                    split_num_items = split_batch_count * self._batch_size
-                    num_items = (
-                        min(num_items, split_num_items * (worker_id + 1))
-                        - split_num_items * worker_id
-                    )
-                    num_evened_items = num_items
-                    start_offset = (
-                        big_batch_count * self._batch_size * self._rank
-                        + min(self._rank * self._batch_size, big_batch_remain)
-                        + self._batch_size
-                        * (
-                            total_batch_count // num_workers * worker_id
-                            + min(worker_id, total_batch_count % num_workers)
-                        )
-                    )
-            else:
-                # Needs to drop uneven batches. As many items as `last_batch`
-                # size will be dropped. It would be better not to let those
-                # dropped items come from the same worker.
-                num_evened_items = big_batch_count * self._batch_size
-                if num_workers > 1:
-                    total_batch_count = big_batch_count
-                    split_batch_count = total_batch_count // num_workers + (
-                        worker_id < total_batch_count % num_workers
-                    )
-                    split_num_items = split_batch_count * self._batch_size
-                    split_item_remain = last_batch // num_workers + (
-                        worker_id < last_batch % num_workers
-                    )
-                    num_items = split_num_items + split_item_remain
-                    num_evened_items = split_num_items
-                    start_offset = (
-                        big_batch_count * self._batch_size * self._rank
-                        + min(self._rank * self._batch_size, big_batch_remain)
-                        + self._batch_size
-                        * (
-                            total_batch_count // num_workers * worker_id
-                            + min(worker_id, total_batch_count % num_workers)
-                        )
-                        + last_batch // num_workers * worker_id
-                        + min(worker_id, last_batch % num_workers)
-                    )
+        total = len(self._item_set)
+        start_offset, num_items, num_output = calculate_range(
```

Comment:
Please align the internal variable and external variable name to avoid confusion.